src/main/java/com/machowina/exception/TicketNotStoppedException.java

```diff
@@ -0,0 +1,15 @@
+package com.machowina.exception;
+
+public class TicketNotStoppedException extends RuntimeException {
+
+	private static final long serialVersionUID = 5058211773026907707L;
+
+	public TicketNotStoppedException() {
```

Comment:
zawsze potrzebujesz dwóch konstruktorów przy wyjątkach?

Reply:
Nie, ale nie byłam pewna którego będę potrzebować podczas dalszego pisania. Powinnam wyrzucić te konstruktory które nie zostały użyte?

Reply:
Tak. A najlepiej nie tworzyć konstruktorów przed tym, jak ich gdzieś indziej używasz ;) Ale to już TDD i głębszy stopień wtajemniczenia.